### ISSUE TITLE
charm build hide_metrics can not be disabled

### ISSUE THREAD

**Issue by mbruzek** [juju/charm-tools#50]
When running `charm build -l DEBUG` I noticed that "hide_metrics" is true.  I tried to turn it to false, using the --hide-metrics flag, but was unable to change the setting.

Shouldn't hide_metrics default to false and --hide-metrics set it to true?  We should fix that.

**Comment by mbruzek** [comment]
Here is the output showing that the default value is true:
```
mbruzek@warhorse:/tmp/trusty/matt$ charm build -l DEBUG
build: Composing into .
build: {
  "INTERFACE_PATH": null, 
  "JUJU_REPOSITORY": null, 
  "LAYER_PATH": null, 
  "_charm": ".", 
  "_name": "matt", 
  "config": "<charmtools.build.config.BuildConfig object at 0x7f96371d7f90>", 
  "deps": "./deps", 
  "force": false, 
  "hide_metrics": true, 
  "interface_service": "http://interfaces.juju.solutions", 
  "log_level": "DEBUG", 
  "output_dir": ".", 
  "repo": "./trusty", 
  "series": "trusty", 
  "target_dir": "./trusty/matt"
}
```

**Comment by mbruzek** [comment]
I was on charm tools version 1.8.0, and this was already fixed in 1.9.2.
with commit: b2ec3b137146e605d609ef8b22a9d7e7632859b5
